homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -148,10 +148,10 @@ def __init__(
 
         self.entity_description = description
         self._attr_unique_id = f"{sensor.sensor_id}-{description.key}"
-        self._attr_name = f"{sensor.location.name} {description.name}"
+        self._attr_name = f"{sensor.name} {description.name}"
```

Comment:
Please see our entity naming guidelines for how to set entity name.

https://developers.home-assistant.io/docs/core/entity#entity-naming

Reply:
Are we talking about a case where the user have multiple devices of exactly the same type? That's a general case and we don't need to consider that. The user can change the name of the device and entities.

Reply:
With this system, a sensor has multiple, I'll call them attributes. The main display that comes with a starter kit has a temperature sensor to measure the indoor temperature. Meanwhile, there's also a thermometer with the same exact attributes. This results in the confusion between entity names. That is the reasoning behind this change.

Reply:
Does the temperature sensor for the indoor temperature and thermometer sensor sit on different devices? Why are the devices called the same, in that case?

Reply:
They sit on different devices, and `sensor.name` would be something the user decides on the LaCrosse View app. What do you mean by that they're called the same?

Reply:
When using our new entity name strategy the device name will be included in the friendly name and entity id. If the devices are named with different names there will be no collision of entity friently names or entity ids, as described in your problem statement above.

Reply:
Gotcha. Working on it.